docs/web-framework-integration.rst

```diff
@@ -0,0 +1,32 @@
+Integration with Python Web Frameworks
+======================================
+
+The Google Cloud Logging library can integrate with Python web frameworks
+`flask <https://flask.palletsprojects.com/>`_ and `django <https://www.djangoproject.com/>`_ to
+automatically populate `LogEntry fields <https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry>`_ 
+`trace`, `span_id`, `trace_sampled`, and `http_request`.
+
+Django
+------
+
+Django integration has been tested to work with each of the Django/Python versions listed `here <https://docs.djangoproject.com/en/5.0/faq/install/#what-python-version-can-i-use-with-django>`_. 
+To enable Django integration, add `google.cloud.logging_v2.handlers.middleware.RequestMiddleware` to the list of `MIDDLEWARE`
+in your settings file. Also be sure to :doc:`set up logging </std-lib-integration>` in your settings file.
```

Comment:
Maybe add a link like this to "settings file"? https://docs.djangoproject.com/en/5.0/topics/settings/